srml/assets/src/lib.rs

```diff
@@ -14,7 +14,149 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
+//! # Assets Module
+//! <!-- Original author of paragraph: @gavofyork -->
+//!
 //! A simple, secure module for dealing with fungible assets.
+//!
+//! Run `cargo doc --package srml-assets --open` to view this module's documentation.
```

Comment:
You don't want to read how to open the documentation, when you already have it opened.